### DISCUSSION TITLE
Icon alignment in the disclosure component

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3429] [Help: Figma] [answered]
hey, I've noticed that the h3 text isn't center aligned with the icon in the disclosure component. Is this an error, or was it intentional?

<img width="1227" alt="Zrzut ekranu 2023-08-21 o 14 02 19" src="https://github.com/twilio-labs/paste/assets/140410476/ffb078c6-04d3-48a0-a1a4-f391d083d467">

**Comment by loreina** [comment]
Hi @marektonik,

Are you in the [Paste Components](https://www.figma.com/file/9QCwINrTkfahYjxnrZnNAl/fix(Disclosure)%3A-adjust-chevron-icon-alignment?type=design&node-id=3015-46646&mode=design) file? The [Disclosure](https://www.figma.com/file/9QCwINrTkfahYjxnrZnNAl/fix(Disclosure)%3A-adjust-chevron-icon-alignment?type=design&node-id=23696-243146&mode=design&t=CSz1OvOmRo8JNVsP-4) component looks accurate for me:

<img width="1125" alt="image" src="https://github.com/twilio-labs/paste/assets/23108291/49d62827-fbd7-4da5-90ae-550fd1237c4a">

**Reply by marektonik** [reply]
@loreina yes, problem for heading 3 is still out there:

<img width="1457" alt="Zrzut ekranu 2023-08-21 o 17 56 51" src="https://github.com/twilio-labs/paste/assets/140410476/5713c82d-8d75-49a7-ac4e-bab1ede2b4b8">

**Reply by loreina** [reply]
Thanks @marektonik, I'll patch this up today and follow up here when that's published!

**Comment by serifluous** [comment, marked as answer]
Hi @marektonik ! We just published the fix.
